### ISSUE TITLE
Remove user from groups after unassigning a tag.

### ISSUE THREAD

**Issue by letsintegreat** [letsintegreat/Tagify#19]
When a user is assigned a tag, we are iterating through the list of groups, and for the ones he is not in yet, we are check if he is now eligible to join the group after the assignment of a new issue, check the code [here](https://github.com/letsintegreat/Tagify/blob/main/lib/screens/all_tags_screen.dart#L168).

Right now we are not removing the user from those groups he is no longer eligible to be in, after un-assignment of any tag. You have to write a similar logic, and check for all the groups he is in, if he is eligible now, remove the user from that group, and do the relevant changes in the firestore database.

Ask to be assigned in the comments below before starting to work on this issue. Stick to the contribution guidelines as mentioned in the README file.

**Comment by mauryakshitij** [comment]
I would like to work on this issue

**Comment by letsintegreat** [comment]
> I would like to work on this issue

All yours.
